examples/Modals/ModalBody.md

```diff
@@ -56,79 +65,76 @@ const [modalVisible, setModalVisible] = useState(false);
           isLegendVisible={false}
         >
           <Checkbox
-          label="Option 1"
-          disabled
-          checked
+            label="Option 1"
+            disabled
+            checked
           />
           <Checkbox
-          label="Option 2"
-          disabled
-          checked
+            label="Option 2"
+            disabled
+            checked
           />
           <Checkbox
-          label="Option 3"
-          disabled
-          checked
+            label="Option 3"
+            disabled
+            checked
           />
           <Checkbox
-          label="Option 4"
-          disabled
-          checked
+            label="Option 4"
+            disabled
+            checked
           />
           <Checkbox
-          label="Option 5"
-          disabled
-          checked
+            label="Option 5"
+            disabled
+            checked
           />
           <Checkbox
-          label="Option 6"
-          disabled
-          checked
+            label="Option 6"
+            disabled
+            checked
           />
           <Checkbox
-          label="Option 7"
-          disabled
-          checked
+            label="Option 7"
+            disabled
+            checked
           />
           <Checkbox
-          label="Option 8"
-          disabled
-          checked
+            label="Option 8"
+            disabled
+            checked
           />
           <Checkbox
-          label="Option 9"
-          disabled
-          checked
+            label="Option 9"
+            disabled
+            checked
           />
           <Checkbox
-          label="Option 10"
-          disabled
-          checked
+            label="Option 10"
+            disabled
+            checked
           />
-          <Checkbox
-          label="Option 11"
-          disabled
-          checked
-          />
-          <Checkbox
-          label="Option 12"
-          disabled
-          checked
+          <TextInput
+            value="test"
+            name="example"
+            id="example"
+            label="Description:"
+            disabled="disabled"
+            onChange={() => {}}
           />
```

Comment:
This example doesn't match up with our `CheckLabel` and `InputLabel` examples. I think we'd want this `TextInput` to be `position={POSITION.TOP}`

Reply:
D'oh. I updated this in 5792c23.